ios/NewExpensify.xcodeproj/project.pbxproj

```diff
@@ -827,7 +828,7 @@
 				COPY_PHASE_STRIP = YES;
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = i386;
```

Comment:
I think this was automatically added by Xcode 14 (and so will likely be introduced in another PR at some point). I can remove it but afaik this simply excludes the 32-bit iOS simulator arch (iOS 10 or lower) and the app's minimum deployment target is iOS 11

Reply:
I am also on Xcode 14, and it does add. I am just not sure if it needs to be pushed, hence @aldo-expensify to check. Not a blocker for me.

Reply:
Yes, since they are not related to what we are doing here, please remove. I'm also getting them in my dev env. I guess at some point we will push them, but not mixed up with this PR.